app/src/main/res/values/dimens.xml

```diff
@@ -217,6 +217,13 @@
     <dimen name="credit_cards_saved_cards_item_margin_start">16dp</dimen>
     <dimen name="credit_cards_saved_cards_item_margin_end">48dp</dimen>
 
+    <!-- Add Credit Card Button Dimensions -->
```

Comment:
Maybe we should just group this with the above and rename `Credit Cards Fragment` to `Credit Cards`